src/native/libs/System.Native/pal_networkstatistics.c

```diff
@@ -401,14 +398,36 @@ int32_t SystemNative_GetActiveTcpConnectionInfos(NativeTcpConnectionInformation*
          xHeadPtr = (struct xinpgen*)((uint8_t*)xHeadPtr + xHeadPtr->xig_len))
     {
         connectionIndex++;
+        NativeTcpConnectionInformation* ntci = &infos[connectionIndex];
         struct xtcpcb* head_xtcpb = (struct xtcpcb*)xHeadPtr;
 
+#ifdef __FreeBSD__
+        struct xinpcb* in_pcb =  &head_xtcpb->xt_inp;
+        if ((in_pcb->inp_vflag & INP_IPV6) == INP_IPV6)
+        {
+            ntci->LocalEndPoint.NumAddressBytes = 16;
+            ntci->RemoteEndPoint.NumAddressBytes = 16;
+            memcpy_s(&ntci->LocalEndPoint.AddressBytes, sizeof_member(IPEndPointInfo, AddressBytes), &in_pcb->inp_inc.inc_ie.ie6_laddr.s6_addr, 16);
+            memcpy_s(&ntci->RemoteEndPoint.AddressBytes, sizeof_member(IPEndPointInfo, AddressBytes), &in_pcb->inp_inc.inc_ie.ie6_faddr.s6_addr, 16);
```

Comment:
In the other occurrences of this pattern, NumAddressBytes are set after the memcpy_s.  Why are they set before here?

Reply:
I don't think the order matter. But I updated it to make it consistent.